documentation/src/main/markdown/releasenotes.md

```diff
@@ -10,11 +10,13 @@
 ### Changed features
 
 * Evicted dependencies in Simple Build Tool(SBT) projects will no longer be included in the Bill of Materials(BoM) generated during the scan.
+* Introduced an optional flag <code>--detect.project.inspector.global.arguments</code> for global flags of Project Inspector. Then, <code>--quite</code> flag and other global flags can be set to this new flag if needed for appropriate pass through to Project Inspector.
 
 ### Resolved issues
 
 * (IDETECT-3111) When scanning SBT projects, "Evicted" dependencies are excluded from the resulting BOM.
 * (IDETECT-3685) Gracefully handled use case when a Poffile.lock file has no PODS or dependencies in the generated dependency graph.
+* (IDETECT-3738) Repositioned the global flags for project inspector invocation.
```

Comment:
Suggested tweak:

* (IDETECT-3738) Repositioned the global flags for inclusion before sub-commands for project inspector invocation.